src/Gui/PreferencePages/DlgSettings3DView.ui

```diff
@@ -122,6 +122,96 @@ in the corner -- in % of height/width of viewport</string>
         </item>
        </layout>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QLabel" name="xAxisLabel">
+          <property name="text">
+           <string>X-axis color</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="Gui::PrefColorButton" name="xAxisColor">
+          <property name="text">
+           <string/>
+          </property>
+          <property name="prefEntry" stdset="0">
+           <cstring>AxisXColor</cstring>
+          </property>
+          <property name="prefPath" stdset="0">
+           <cstring>View</cstring>
+          </property>
+         </widget>
```

Comment:
don't you need to set a default value here too? Otherwise if I'm not mistaken the default color from the widget (white?) will be set when you first open the preference page.

Reply:
I actually don't know. I tried setting the default it in the .cfg file... Seems like some preference entries have their default implemented in `.ui` and in `.cfg...`? On top of that, ViewParams.h have it's own implementation of defaults 😕

Reply:
That is the case, unfortunately: https://github.com/FreeCAD/FreeCAD/issues/14919